docs/workflows/oa_web.md

```diff
@@ -1,6 +1,6 @@
 # Open Access Website Workflow
-A workflow that generates the data for the COKI Open Access Website (still in progress), copies the data to the website
-static folder, builds the website and then deploys it to Cloudflare.
+A workflow that generates the data for the COKI Open Access Website and uploads the data to a Google Cloud Storage
+bucket, which is used to build the Next.js web application.
 
```

Comment:
```suggestion
Each time the workflow runs successfully a repository dispatch event is created, this event triggers the Github actions responsible for building the web application
```
Or something along those lines, I think it might be useful to add a sentence to clarify that the github actions are triggered from this workflow. and that those actions build the web application.

Reply:
Thanks, I've added that